### PR TITLE
add API that allows admins to decode galaxy objects' ids

### PR DESCRIPTION
**Pull request by martenson**
ping @erasche I think you requested this in the past too?

### CODE REVIEW

File: test/api/test_configuration.py [hunk 1]
```diff
@@ -34,6 +41,22 @@ def test_admin_user_configuration(self):
...
+        decoded_library_id = self.security.decode_id(new_lib["id"])
+        assert decoded_library_id == response_id
+        # fake valid folder id by prepending F
+        valid_encoded_folder_id = 'F' + new_lib["id"]
+        folder_decode_response = self._get("configuration/decode/" + valid_encoded_folder_id, admin=True)
+        folder_response_id = folder_decode_response.json()["decoded_id"]
+        assert decoded_library_id == folder_response_id
+
+    def test_normal_user_decode_id(self):
+        decode_response = self._get("configuration/decode/badhombre", admin=False)
```
Comment (hexylena) [inline test/api/test_configuration.py:57]: (I trust the `@require_admin` flag, but) shouldn't we test decoding a valid ID with admin=False?

**Comment by hexylena** [comment]
sounds like something I'd request :) LGTM :+1:

File: test/api/test_configuration.py (hunk 1 above)
Comment (martenson) [reply test/api/test_configuration.py:57]: this technically tests that, since the response won't be 403

**Comment by jmchilton** [comment]
Thanks Martin - I really just wanted to have a plan for how to do this but an actual implementation is great. Thanks for putting this together!

**Pull request merged**